httpcore5-h2/src/main/java/org/apache/hc/core5/http2/impl/nio/bootstrap/H2ServerBootstrap.java

```diff
@@ -522,7 +523,7 @@ public HttpAsyncServer create() {
         final ServerH2StreamMultiplexerFactory http2StreamHandlerFactory = new ServerH2StreamMultiplexerFactory(
                 httpProcessor != null ? httpProcessor : H2Processors.server(),
                 handlerFactory,
-                h2Config != null ? h2Config : H2Config.DEFAULT,
+                h2Config != null ? h2Config : H2Config.custom().setPushEnabled(false).build(),
```

Comment:
@CoolTomatos Could you make a static private variable out of it? Looks good otherwise.